### PR TITLE
Notifier test copy and bulk delete

### PR DESCRIPTION
**Pull request by MilanPospisil**

### CODE REVIEW

File: cypress/e2e/awx/administration/notifiers/notifiersListView.cy.ts [hunk 1]
```diff
@@ -83,37 +80,71 @@ describe('Notifications: List View', () => {
     //Assert the behavior in the UI following the test action
   });
 
-  it.skip('can copy a Notification and assert that the copy action completed successfully', () => {
+  it('can copy a Notification and assert that the copy action completed successfully', () => {
     //Utilize a notification of any type created in the beforeEach hook
     //Assert the existence of the notification before copy
     //Assert the copy action
     //Assert the existence of the copied notification as well as the original
-    cy.navigateTo('awx', 'notification-templates');
```
Comment (vishsanghishetty) [inline cypress/e2e/awx/administration/notifiers/notifiersListView.cy.ts:91]: this line needs to be added back. once the login happens, our next action is navigating to notifiers

File: cypress/e2e/awx/administration/notifiers/notifiersListView.cy.ts [hunk 2]
```diff
@@ -83,37 +80,71 @@ describe('Notifications: List View', () => {
...
-    cy.get('[data-cy="checkbox-column-cell"]').within(() => {
-      cy.get('input').click();
-    });
-    cy.clickToolbarKebabAction('delete-selected-notifiers');
-    cy.getModal().within(() => {
-      cy.get('#confirm').click();
-      cy.clickButton(/^Delete notifiers/);
-      cy.contains(/^Success$/);
-      cy.clickButton(/^Close$/);
+    cy.getByDataCy(`awx-notification-templates`).click({ force: true });
```
Comment (vishsanghishetty) [inline cypress/e2e/awx/administration/notifiers/notifiersListView.cy.ts:88]: please use `cy.navigateTo('awx', 'awx-notification-templates')`

File: cypress/e2e/awx/administration/notifiers/notifiersListView.cy.ts [hunk 3]
```diff
@@ -83,37 +80,71 @@ describe('Notifications: List View', () => {
...
+      cy.filterTableByMultiSelect('name', [`${notificationTemplate.name} @`]);
+      cy.get('[data-cy="checkbox-column-cell"]').within(() => {
+        cy.get('input').click();
+      });
+      cy.clickToolbarKebabAction('delete-selected-notifiers');
+      cy.getModal().within(() => {
+        cy.get('#confirm').click();
+        cy.clickButton(/^Delete notifiers/);
+        cy.contains(/^Success$/);
+        cy.clickButton(/^Close$/);
```
Comment (vishsanghishetty) [inline cypress/e2e/awx/administration/notifiers/notifiersListView.cy.ts:108]: after this line, add `cy.clickButton(/^Clear all filters$/);`

File: cypress/e2e/awx/administration/notifiers/notifiersListView.cy.ts [hunk 4]
```diff
@@ -83,37 +80,71 @@ describe('Notifications: List View', () => {
...
   });
 
-  it.skip('can bulk delete a Notification and assert deletion', () => {
+  it('can bulk delete a Notification and assert deletion', () => {
     //Utilize notification created in the beforeEach block
     //create an additional notification in this test for the purposes of bulk delete
     //Assert the presence of the items before deletion
     //Assert the deletion
+
+    cy.getByDataCy(`awx-notification-templates`).click({ force: true });
```
Comment (vishsanghishetty) [inline cypress/e2e/awx/administration/notifiers/notifiersListView.cy.ts:119]: please use `cy.navigateTo('awx', 'awx-notification-templates')`

File: cypress/e2e/awx/administration/notifiers/notifiersListView.cy.ts [hunk 5]
```diff
@@ -83,37 +80,71 @@ describe('Notifications: List View', () => {
...
+          .eq(1)
+          .within(() => {
+            cy.get('input').click();
+          });
+        cy.clickToolbarKebabAction('delete-selected-notifiers');
+        cy.getModal().within(() => {
+          cy.get('#confirm').click();
+          cy.clickButton(/^Delete notifiers/);
+          cy.contains(/^Success$/);
+          cy.clickButton(/^Close$/);
```
Comment (vishsanghishetty) [inline cypress/e2e/awx/administration/notifiers/notifiersListView.cy.ts:141]: add `cy.clickButton(/^Clear all filters$/);` after line 141

**Review by vishsanghishetty — APPROVED** [review]

File: cypress/e2e/awx/administration/notifiers/notifiersListView.cy.ts (hunk 2 above)
Comment (MilanPospisil) [reply cypress/e2e/awx/administration/notifiers/notifiersListView.cy.ts:88]: This does not work, the PRs are failing that element is covered, not sure why...

Comment (MilanPospisil) [reply cypress/e2e/awx/administration/notifiers/notifiersListView.cy.ts:88]: I will try to merge this and later solve this problem, so it does not block the test.

**Pull request merged**